app/Providers/AppServiceProvider.php

```diff
@@ -206,6 +206,182 @@ function.
                     'legal-image-licensing' => '/image-licensing',
                     'legal-saic' => 'https://www.saic.edu',
                 ],
+                'primaryNav' => [
+                    [
+                        'name' => 'Visit',
+                        'description' => 'Find all the information you need&mdash;plus helpful tips&mdash;to plan your visit',
+                        'image' => url('/images/navigation-thumbnail-visit.jpg'),
+                        'children' => [
+                            [
+                                'name' => 'Hours',
+                                'url' => route('visit') . '#hours',
+                            ],
+                            [
+                                'name' => 'Admission',
+                                'url' => route('visit') . '#admission',
+                            ],
+                            [
+                                'name' => 'Plan Your Visit',
+                                'url' => route('visit') . '#plan-your-visit',
+                                'children' => [
+                                    [
+                                        'name' => 'Museum map',
+                                        'url' => '/visit/explore-on-your-own/museum-floor-plan',
+                                    ],
+                                    [
+                                        'name' => 'Free daily tours',
+                                        'url' => route('events', ['audience' => 3, 'type' => 6]),
+                                    ],
+                                    [
+                                        'name' => 'What to see in an hour',
+                                        'url' => route('highlights.show', ['id' => 3, 'slug' => 'what-to-see-in-an-hour']),
+                                    ],
+                                    [
+                                        'name' => 'Shopping and dining',
```

Comment:
Make "and" an ampersand.

Reply:
Josh checked with Comms, and they only want the capitalization changed.